### PR TITLE
Revert "demo"

### PR DESCRIPTION
**Pull request by Anirudh906**
Reverts prashantnk/spring-security-demo#1

### CODE REVIEW

File: src/main/java/com/springsecurity/demo/model/MUser.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 public class MUser {
 	@NotNull(message = "is required ! ")
-	@Size(min = 1, message = "atleast of 5 characters")
```
Comment (prashantnk) [inline src/main/java/com/springsecurity/demo/model/MUser.java:8]: this is demo comment

**Review by prashantnk — COMMENTED** [review]
this is demo comment 2

File: src/main/java/com/springsecurity/demo/model/MUser.java (hunk 1 above)
Comment (Anirudh906) [reply src/main/java/com/springsecurity/demo/model/MUser.java:8]: reply

Comment (Anirudh906) [reply src/main/java/com/springsecurity/demo/model/MUser.java:8]: this should be 2

**Pull request closed without merging**